### ISSUE TITLE
Docking does not function as expected on Linux

### ISSUE THREAD

**Issue by DaveJarvis** [Drombler/drombler-fx#264]
# Overview

Drombler dockable example does not work

# Replicate

1. Download [drombler.zip](https://github.com/Drombler/drombler-fx/files/5406227/drombler.zip)
1. Extract files.
1. Open project in IntelliJ.
1. Run project.

# Expected

The application allows tabs to be docked alongside existing tabs using drag and drop functionality.

# Actual

Numerous issues:

* Tabs cannot be docked.
* Closing a tab (e.g., the left tab) then using Window menu to re-open the tab partially succeeds: the tab itself is not rendered correctly until the panel is resized---the tab appears 1px wide.

# Environment

JavaFX 11, Liberica JDK 14, Gradle 6.6.1, Linux (X11/XFCE).

**Comment by puce77** [comment]
Regarding the drag and drop functionality I thinkt this is a duplicate of #22 and #23 which in turn are blocked by a long-time open issue of JavaFX https://bugs.openjdk.java.net/browse/JDK-8092098

**Comment by puce77** [comment]
Regarding the rendering issue, I will have to analyze it.

**Comment by ghost** [comment]
FWIW https://github.com/xylo/DraggableTabs works around the JDK bug to provide draggable tabs.

See also the solutions listed in https://github.com/DaveJarvis/keenwrite/issues/65#issuecomment-673773740

**Comment by puce77** [comment]
I could not reproduce this issue on Kubuntu 18.04.5 LTS with Java 8/ JavaFX 8.

Please note that Drombler Commons v1.x and Drombler FX v1.x are Java 8 based and thus JavaFX 8 based.
The build script uses JavaFX 14 though (which I excluded for testing as JavaFX is bundled with Oracle JDK 8).

There has already be done a lot of work to migrate Drombler FX to Java / JavaFX >=11 (https://github.com/Drombler/drombler-fx/issues/172) but unfortunately Apache Felix isn't there yet: https://issues.apache.org/jira/browse/FELIX-6287

Please try:

- Run the sample with Java 8/ JavaFX 8 and see if this bug exists. Provide an updated sample if the bug exists there.
- Run the sample with a higher Java / JavaFX version but use Drombler Commons 2.0-SNAPSHOT (use Drombler Commons Dependencies for the test, not Drombler FX) (see the README for more infos regarding SNAPSHOT dependencies). Provide an updated sample if the bug exists there.

**Comment by ghost** [comment]
@puce77, I cannot run this with JavaFX 8 because [my application](https://github.com/DaveJarvis/keenwrite) is developed using JavaFX 11.

Unfortunately, by the time Drombler works with JavaFX 11 in both modular and non-modular form, I'll have likely integrated a [different solution](https://github.com/xylo/DraggableTabs).
